batch/batch/driver/job.py

```diff
@@ -201,23 +207,25 @@ async def mark_job_creating(
 
     log.info(f'mark job {id} creating')
 
-    try:
-        rv = await db.execute_and_fetchone(
-            '''
+    @transaction(db)
+    async def mjc_and_add_attempt_resources(tx: Transaction) -> dict:
+        try:
+            rv = await tx.execute_and_fetchone(
+                '''
 CALL mark_job_creating(%s, %s, %s, %s, %s);
 ''',
-            (batch_id, job_id, attempt_id, instance.name, start_time),
-        )
-    except Exception:
-        log.info(f'error while marking job {id} creating on {instance}')
-        raise
+                (batch_id, job_id, attempt_id, instance.name, start_time),
+            )
+        except Exception:
+            log.info(f'error while marking job {id} creating on {instance}')
+            raise
+        await add_attempt_resources(tx, batch_id, job_id, attempt_id, resources)
+        return rv
+    rv = mjc_and_add_attempt_resources()  # pylint: disable=no-value-for-parameter
 
-    log.info(rv)
```

Comment:
I'm not sure why this was here. Neither MJC nor MJS log their return value without any context.